parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -28,13 +28,20 @@ spec:
         - "--tls-cert-file=/etc/kubernetes/certs/apiserver.crt"
         - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
+        - "--requestheader-client-ca-file=/etc/kubernetes/certs/proxy-ca.crt"
```

Comment:
should we be wrapping these in {{if .OrchestratorProfile.KubernetesConfig.EnableAggregatedAPIs}}  ?

Reply:
That is done in `parts/kubernetesmastercustomdata.yml`, I was under the impression that I wouldn't be able to do it explicitly in this file. Happy to be wrong about that.

Reply:
probably doesn't unless we have this running through the template engine which I don't think we do. We could do sed or the like but that would be ugly. Since this is working I guess we can leave.